projects/packages/forms/src/blocks/contact-form/class-contact-form-block.php

```diff
@@ -257,6 +257,32 @@ public static function register_child_blocks() {
 				'render_callback' => array( Contact_Form_Plugin::class, 'gutenblock_render_field_checkbox_multiple' ),
 			)
 		);
+
+		/**
+		 * The block 'jetpack/field-checkbox-multiple' is a wrapper block.
+		 * Styles must be registered
+		 * so that they are available to be overridden by the theme or global styles.
+		 * Form field blocks define the block style via the settings in their index.js files.
+		 * A follow up issue is to update them to use block.json files, which can be reused
+		 * in both JS and PHP block registration.
+		 */
+		register_block_style(
+			'jetpack/field-checkbox-multiple',
```

Comment:
What's the minimum WP version for Jetpack? 

Recent versions should support passing multiple block names in an array. It's neither here nor there but would save the repeated `register_block_style` calls.

Reply:
Looks like the tests expect at least 6.6... so it should be find to use `string[]`. Thanks for the nudge.